digitalstrategie/assets/scss/components/_books.scss

```diff
@@ -1,4 +1,19 @@
+// Books (Grünbuch and Weißbuch) specific overwrites
```

Comment:
But this overwrites all headlines, doesn't it? 
So, either add utility classes with what they do
```
.u-font-size-h2 {
    font-size: $h2-font-size;
}
```
(https://github.com/liqd/digitalstrategie/blob/40da7a3f22848688e33a98fedd1f43b85a586cc6/digitalstrategie/assets/scss/_variables.scss#L197)

Alright, while looking up the bootstrap utility for the font weight, I found that there are also some for the font sizing! So, you could replace all of the headline styling by using bootstraps utility classes directly in the template:
`fw-normal`, `fs-2`, `fs-5` and `mt-0`.

https://getbootstrap.com/docs/5.0/utilities/text/

Or instead of utilies, you could also use specific class names like `books__h1`